guides/Expo Universal Module Infrastructure.md

```diff
@@ -13,11 +13,11 @@ This guide explains the standard configuration and tools for working on universa
 
 # Generating a new universal module using `expo-cli` command
 
-`expo-cli` has specific command that would generate universal module that support TypeScript!
+`expo/tools` has specific command that would generate universal module that support TypeScript!
 Run:
 
-- `expo generate-module [new module directory]`
-  - optional `[new module directory]` parameter lets you specify module name (e.g. `expo generate-module expo-test-module` would create `expo-test-module`. If ommited, the script will prompt you about it.
+- `yarn et cu --name [new module name]` inside the `tools` directory
```

Comment:
```suggestion
- `expotools create-unimodule --name [new module name]` inside the Expo workspace
```

I'm not aware of context of those changes, but in docs or guides we tend to avoid the commands abbreviations.

I'm also not sure about `yarn` in there, since afaik if you have the Expo Tools have been built and linked correctly, adding `yarn` before the actual command should not be needed.